src/rodeo/member.rs

```diff
@@ -0,0 +1,160 @@
+use crate::{item::Item, util::MD5Hash};
+
+use super::{
+  goat::GoatRodeoCluster, goat_synth::GoatSynth, goat_trait::GoatRodeoTrait, index::ItemOffset,
+};
+use anyhow::Result;
+use std::{path::PathBuf, sync::Arc};
+use tokio::sync::mpsc::Receiver;
+use tokio_util::either::Either;
+
+#[derive(Debug, Clone)]
```

Comment:
Should `GoatSynth`, `GoatRodeoCluster` and `HerdMember` each implement a trait for `name` `offset_from_pos` and `item_from_item_offset`?